RubeesEat/wwwroot/css/site.css

```diff
@@ -0,0 +1,172 @@
+/*source: https://www.joshwcomeau.com/css/custom-css-reset/*/
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+* {
+    margin: 0;
+    font-family: "Metric Regular", Arial, Tahoma, sans-serif;
+    text-align: center;
```

Comment:
Please no. This is not a good fix for "I want most of my stuff aligned to the middle". This rule applies to every element and thus makes styling very hard if you don't want `text-align: center`. It also disables inheritance completely which is not good.